src/tests/ftest/interoperability/updown_grade_8svr.py

```diff
@@ -0,0 +1,32 @@
+"""
```

Comment:
This test and config looks similar to `updown_grade.py`. Since these are manual tests anyway, couldn't the same `updown_grade.py` be ran with a slightly different config?
FYI that `launch.py` has an option to supply an extra config for just this purpose:
```
$ ./launch.py --help | grep -A3 extra_yaml
  -e EXTRA_YAML, --extra_yaml EXTRA_YAML
                        additional yaml file to include with the test yaml
                        file. Any entries in the extra yaml file can be used
                        to replace an existing entry in the test yaml file.
```

Reply:
ok. I will remove this .py file, and use "-e updown_grade_8svr.yaml".